README.md

```diff
@@ -63,6 +63,7 @@ Config parameter details:
  * `forceAuthn`: if set to true, the initial SAML request from the service provider specifies that the IdP should force re-authentication of the user, even if they possess a valid session.
  * `skipRequestCompression`: if set to true, the SAML request from the service provider won't be compressed.
  * `authnRequestBinding`: if set to `HTTP-POST`, will request authentication from IDP via HTTP POST binding, otherwise defaults to HTTP Redirect
+ * `authnRequestPostForm`: if `HTTP-POST` is used for `authnRequestBinding`, you can use this to customize the form which will be posted. Pass a function which processes an array of name/value objects and returns your custom form as a string.
```

Comment:
Going with comment from below, function signature should be added here for clarity as to usage